src/index.js

```diff
@@ -1,6 +1,9 @@
 import dotenv from 'dotenv';
 dotenv.config();
 
+import consoleStamp from 'console-stamp';
+consoleStamp(console);
```

Comment:
> Not super fan of patching the `console` with an RC dependency; I'm wondering if we really need to do that for the few calls to `console.log` we have, vs simply using `console.log(new Date().toISOString(), 'message')` on the two or three instances where it's useful.

Reply:
I understand but I find it very useful to have this information added automatically and I don't want to do that manually every time I call `console.log`, maybe I can look for another lib if you prefer?

Reply:
Or, I can use the version 2.9.0 and not the 3.0.0RC?

Reply:
I chose the last option